platform/security/core/security-core-services/src/main/java/ddf/security/http/impl/HttpSessionFactory.java

```diff
@@ -44,4 +47,8 @@ public synchronized HttpSession getOrCreateSession(HttpServletRequest httpReques
     }
     return session;
   }
+
+  public void setExpirationTime(int expirationTime) {
+    this.expirationTime = expirationTime;
```

Comment:
✏️ I haven't looked, but assume that somewhere beneath this - on the session itself - we perform the logic to check the provided value and if it's less than two minutes, use the default. I think it might be better to prevent setting it to less than two minutes right here.

Two int consts in this class - `MINIMUM_EXP` and `DEFAULT_EXP` could be defined as 2 and 30; this setter could check the value of `expirationTime` and set to `DEFAULT_EXP` if the provided is less than 2.

Reply:
Going to hold off on doing this, just because we are about to remove the reason why you can't set it less than 2 minutes in the first place. We'll just clean up the metatype descriptions when we're done.